apps/src/applab/applab.js

```diff
@@ -820,7 +821,13 @@ Applab.init = function(config) {
           Applab.runButtonClick();
         });
       }
-    });
+    })
+    .then(
+      () =>
+        config.isJavabuilderConnectionTestEnabled &&
```

Comment:
This could possibly leverage the work in https://github.com/code-dot-org/code-dot-org/pull/43950...

Reply:
I was looking for something like this when I was doing this work and was excited to see Dayne's post yesterday! Since we're going to revert this at some point relatively soon anyway, I don't think it's worth waiting for his PR to make its way to through review and rewriting this to use it. LMK if you disagree.